test/index-react-native.test.ts

```diff
@@ -0,0 +1,100 @@
+import { FiatConnectClient, SiweClient } from '../src/index-react-native'
+import { Network } from '@fiatconnect/fiatconnect-types'
+import * as siwe from 'siwe'
+import 'jest-fetch-mock'
+import CookieManager, { Cookies } from '@react-native-cookies/cookies'
+
+// work around from
+// https://github.com/aelbore/esbuild-jest/issues/26#issuecomment-968853688 for
+// mocking siwe packages
+jest.mock('siwe', () => ({
+  __esModule: true,
+  // @ts-ignore
+  ...jest.requireActual('siwe'),
+}))
+
+const mockSetCookies: Cookies = {
+  session: {
+    name: 'session',
+    value: 'session-val',
+  },
+}
+
+jest.mock('@react-native-cookies/cookies', () => {
+  return {
+    get: jest.fn(),
+  }
+})
+
+describe('FiatConnect SDK node', () => {
```

Comment:
```suggestion
describe('FiatConnect SDK react-native', () => {
```
also needs to be updated in index-browser.test.ts